### PR TITLE
add ridgeline transition functions

### PR DESCRIPTION
**Pull request by collnell**
Changes made:
-----------
Adding some transition code that will  be used to change the ridge plot between a couple different layouts on toggle, said toggle not yet implemented. Added some small style edits - avoiding double headers, shrinking some text size, adding some spice

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/components/SectionTitle.vue [hunk 1]
```diff
@@ -6,9 +6,9 @@
       :style="chaptersVars"
     >
       <div class="chapterTitle">
-        <slot name="chapterTitle">
+     <!--    <slot name="chapterTitle">
           Chapter Title
-        </slot>
+        </slot> -->
       </div>
```
Comment (mwernimont) [inline src/components/SectionTitle.vue:12]: Why are we commenting out the chapter title slot?

Comment (mwernimont) [reply src/components/SectionTitle.vue:12]: This effects each chapter title as its the main component we import.

Comment (collnell) [reply src/components/SectionTitle.vue:12]: I was trying out what it looks like without the extra title but didn't mean to commit it. One thing about the page now is that there are a lot of titles and then double titles each text section. I  don't think it adds a lot to have two titles in series that are very similar, and the smaller titles are more informative. I'm not sure the best solution, but it does look naked without the chapter title slot.

Comment (mwernimont) [reply src/components/SectionTitle.vue:12]: Well my two cents would be to have the section title as the image title and then just forgo the second title and just start with the explanation text.  Its how we have done it in the past, but that could be good reason not to depending how you view our old vizzies. lol.  But, either way let me know and tweaks can be made to any component that we deem tweakable.

Comment (collnell) [reply src/components/SectionTitle.vue]: Yeah I think that could work. We definitely need to keep some  of the subtitles within each section to establish takeaway points as we go along, but if we take away the first subtitle in each section that'll resolve the double header situation. PR  edited.

**Review by mwernimont — APPROVED** [review]

**Pull request merged**